apps/demos/src/app/rx-angular-pocs/cdk/render-aware/render-aware.ts

```diff
@@ -39,34 +51,33 @@ export function createRenderAware<U>(cfg: {
   getCdRef: (k: RxNotification<U>) => ChangeDetectorRef;
   getContext: (k?: RxNotification<U>) => any;
 }): RenderAware<U | undefined | null> {
-
   const strategyName$ = new ReplaySubject<Observable<string>>(1);
   const strategyHandling$ = strategyHandling(
     cfg.defaultStrategyName,
     cfg.strategies
   );
   const templateTriggerSubject = new Subject<Observable<RxNotification<U>>>();
-  const templateTrigger$ = templateTriggerSubject.pipe(
-    mergeAll()
-  );
+  const templateTrigger$ = templateTriggerSubject.pipe(mergeAll());
 
   const observablesFromTemplate$ = new ReplaySubject<Observable<U>>(1);
-  const renderingEffect$ =
+  const renderingEffect$ = connectable(
     observablesFromTemplate$.pipe(
-      map(o => isObservable(o) ? o : of(o)),
+      map((o) => (isObservable(o) ? o : of(o))),
       distinctUntilChanged(),
       switchAll(),
       distinctUntilChanged(),
       rxMaterialize(),
       mergeWith(templateTrigger$ || EMPTY),
-      /*observeTemplateByNotificationKind(cfg.templateObserver),
-      applyStrategy(strategy$, cfg.getContext, cfg.getCdRef),*/
-      catchError(e => {
+      catchError((e) => {
         console.error(e);
         return EMPTY;
-      }),
-      publishReplay()
-    );
+      })
+    ),
+    {
+      connector: () => new ReplaySubject(),
```

Comment:
Is it intended here to have no buffer size limit? @hoebbelsB

Reply:
you can safely remove those pocs. Let's not try to fix something that has no value